ci-operator/config/openshift/aws-load-balancer-controller/openshift-aws-load-balancer-controller-main.yaml

```diff
@@ -25,12 +25,12 @@ promotion:
 releases:
   initial:
     integration:
-      name: "4.12"
+      name: "4.14"
```

Comment:
Should this be 4.15 now? Just like https://github.com/openshift/release/pull/45662